tests/data/packages/TopoRequires4/setup.cfg

```diff
@@ -0,0 +1,6 @@
+[metadata]
```

Comment:
at the pace you're going, you might mave `[metadata]` support sooner, than later, but one step at a time.   

this kind of stuff creates confusion.   my 2c is to take it  out.